Makefile

```diff
@@ -32,6 +32,7 @@ vendor:
 	dep ensure
 
 signalfx-agent: templates
+	echo "building SignalFx agent for operating system: $(GOOS)"
```

Comment:
Make the dev image's environment have `GOOS` to `linux` when run in the `run-dev-image` target, so that this message will be complete in the dev image.